V1Documentation.md

```diff
@@ -609,6 +502,55 @@ In Chrome, if the origin script tag and remote domain do not meet these requirem
 
 Other browsers may send on a best-effort basis (version dependent) if some data is available but potentially without a useful stacktrace. The provider will cancel the send if no data is available.
 
+##### Browser behaviour
+
+Depending on what browser your users are running, the above properties may or may not have an effect. This sums up the situation as of writing:
+
+* Chrome 30+
+* Firefox 13+
+* Opera 12.50+
+* Safari (at least 6+)
+
+In these browsers, if the script attribute is present, the HTTP header will need to be also present, otherwise the script will be blocked.
+
+Firefox has additional behavior for RuntimeErrors. These will be provided to window.onerror regardless of the two properties, as these aren’t considered a security risk. SyntaxErrors, however, will be blocked in both Gecko and WebKit browsers, if crossorigin is present but the associated cross-origin domain lacks the header.
+
+* Internet Explorer <= 10
+
+Errors will be reported with all available data in IE 10 and below.
+
+* Internet Explorer 11+
+
+#### Limitations of stack trace data
+
+Due to browser API and security limitations, in cases where the message is 'Script error', only one stack trace frame may be present. In this scenario, the line number may not reflect the actual position where the original error was thrown.
+
+For more information, check out this blog post on [CORS requirements for Script Errors].
+
+[CORS requirements for Script Errors]: https://raygun.io/blog/2015/05/fix-script-error-and-get-the-most-data-possible-from-cross-domain-js-errors/
+
+## AngularJS
+
+You can hook failed Ajax requests with $http in AngularJS by providing an Interceptor that sends to Raygun on error. One possible simple implementation:
+
+```javascript
+$httpProvider.interceptors.push(function($q, dependency1, dependency2) {
+  return {
+   'requestError': function(rejection) {
+       Raygun.send(new Error('Failed $http request', rejection));
```

Comment:
This looks like the same as the current docs, think we decided on `Raygun.send(new Error('msg'), { rejection: rejection });`